### PR TITLE
added onERC721Received to Executor

### PR DESCRIPTION
**Pull request by 0xDEnYO**

### CODE REVIEW

File: src/Periphery/Executor.sol [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { LibAsset } from "../Libraries/LibAsset.sol";
 import { ILiFi } from "../Interfaces/ILiFi.sol";
 import { IERC20Proxy } from "../Interfaces/IERC20Proxy.sol";
 import { TransferrableOwnership } from "../Helpers/TransferrableOwnership.sol";
+import { IERC721Receiver } from "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
```
Comment (ezynda3) [inline src/Periphery/Executor.sol:11]: Is there any reason to actually import this?

File: src/Periphery/Executor.sol [hunk 2]
```diff
@@ -261,6 +262,20 @@ contract Executor is ILiFi, ReentrancyGuard, TransferrableOwnership {
         return balances;
     }
 
+    /// @dev Will be called as part of an ERC721 token transfer to this contract (see IERC721Receiver for more details)
+    /// @param operator The address of the tx initiator
+    /// @param from The address from which the ERC721 token was transferred to this contract
+    /// @param tokenId The ID of the token that was transferred
+    /// @return uint256[] Array of token balances.
```
Comment (satoshi-lifi) [inline src/Periphery/Executor.sol:269]: return value is not balances(uint256[]) but selector(bytes4)

File: src/Periphery/Executor.sol (hunk 1 above)
Comment (0xDEnYO) [reply src/Periphery/Executor.sol:11]: for explicity purpose and the comments in the file I would tend to include it. But I forgot to let the Executor inherit from it, which I changed now.

File: src/Periphery/Executor.sol (hunk 2 above)
Comment (0xDEnYO) [reply src/Periphery/Executor.sol]: updated

**Pull request merged**